app/src/deploys/deploys.module.ts

```diff
@@ -1,9 +1,13 @@
-import { Module } from "@nestjs/common";
+import { CacheModule, Module } from "@nestjs/common";
+import { NODE_CACHE_LIMIT } from "src/config";
 
 import { DeploysController } from "./deploys.controller";
 import { DeploysService } from "./deploys.service";
 
 @Module({
+  imports: [
+    CacheModule.register({ ttl: Number.MAX_VALUE, max: NODE_CACHE_LIMIT }),
```

Comment:
Do we want a near infinite time to live?

Reply:
Sure do, we'd like to be the ones to manage clearing/updating the cache.

Reply:
Ahh okay okay -- totally makes sense! 👍